services/core-api/app/api/now_applications/models/now_application_progress.py

```diff
@@ -16,6 +16,7 @@ class NOWApplicationProgress(Base, AuditMixin):
 
     application_progress_status_code = db.Column(db.String, nullable=False)
     start_date = db.Column(db.DateTime, nullable=False)
+    end_date = db.Column(db.DateTime)
```

Comment:
This should be changed to `Date` to match the type at the data level. But I think this should remain as `DateTime` and the data type should be changed to be a timestamp (see comment above).